### PR TITLE
Listing node from grid proxy instead of graphql

### PR DESCRIPTION
**Pull request by maayarosama**
### Description
Adjusted the portal section to fetch the nodes from the grid proxy instead of graphql.
### List of related issues
#277

### CODE REVIEW

File: src/portal/views/Farms.vue [hunk 1]
```diff
@@ -279,6 +279,7 @@ import {
   setFarmPayoutV2Address,
 } from "../lib/farms";
 import { StrKey } from "stellar-sdk";
+import config from "../config";
```
Comment (rawdaGastan) [inline src/portal/views/Farms.vue]: is config used in the file?

File: src/portal/lib/farms.ts [hunk 2]
```diff
@@ -216,7 +213,7 @@ export async function addNodePublicConfig(
   address: string,
   api: { tx: { tfgridModule: { addNodePublicConfig: (arg0: any, arg1: any, arg2: any) => { (): any; new(): any; signAndSend: { (arg0: any, arg1: { signer: Signer; }, arg2: any): any; new(): any; }; }; }; }; },
   farmID: number,
-  nodeID: number,
+  nodeId: number,
```
Comment (rawdaGastan) [inline src/portal/lib/farms.ts]: why did you change this?

**Review by rawdaGastan — CHANGES_REQUESTED** [review]

File: src/portal/views/Farms.vue (hunk 1 above)
Comment (maayarosama) [reply src/portal/views/Farms.vue]: Rrmoved The unused import

File: src/portal/lib/farms.ts (hunk 2 above)
Comment (maayarosama) [reply src/portal/lib/farms.ts]: No specific reason, I changed it back as it doesn't affect anything.

**Review by rawdaGastan — DISMISSED** [review]

**Review by rawdaGastan — COMMENTED** [review]
When no farm ids given, all nodes are listed

**Review by rawdaGastan — APPROVED** [review]
